app/assets/javascripts/discourse/tests/acceptance/composer-test.js

```diff
@@ -75,6 +75,7 @@ acceptance("Composer", function (needs) {
         valid_groups: ["staff"],
         mentionable_groups: [{ name: "staff", user_count: 30 }],
         cannot_see: [],
+        cannot_see_groups: [],
```

Comment:
Based on the server's response, I believe this should be an empty object and not an empty array.

Reply:
Yes, that's correct and covered by the refactor.